### ISSUE TITLE
Accessibility: Goto line does not announce that focus moved to a particular place in the file

### ISSUE THREAD

**Issue by xirzec** [Microsoft/monaco-editor#486]
monaco-editor version: 0.8.3
Browser: Edge
OS:  Win10
Steps or JS usage snippet reproducing the issue:

1. Open Windows Narrator
1. Invoke Ctrl+G to go to line
1. Type a line number and hit enter
1. Focus moves to the editor, but there is no announcement for focus moving back to the editor. 

Expected: Some announcement like 'focus moved to line 10 of editor'

In general when we move focus (such as go to defintion) we should announce where focus has been moved to before starting to read the editor contents again.

**Comment by bpasero** [comment]
> In general when we move focus (such as go to defintion) we should announce where focus has been moved to before starting to read the editor contents again.

I agree and I think this is not something each component should implement that changes selection in the editor (like goto line, goto symbol). I feel like a more general approach is needed where the editor announces something as soon as it gets focus back with a changed selection.
